packages/react-strict-dom/tests/html-test.native.js

```diff
@@ -256,6 +259,37 @@ describe('<html.*>', () => {
       expect(secondSpanInner.props.style.lineHeight).toBe(2 * 20);
     });
 
+    test('inherited styles for auto-fix of raw strings', () => {
+      const tokens = css.defineVars({
+        color: 'red',
+        fontSize: '1em',
+        lineHeight: 1
+      });
+      const theme = css.createTheme(tokens, {
+        color: 'green',
+        fontSize: '2em',
+        lineHeight: 3
+      });
+
+      const styles = css.create({
+        root: {
+          color: tokens.color,
+          fontSize: tokens.fontSize,
+          lineHeight: tokens.lineHeight
+        },
+        text: {
+          fontSize: tokens.fontSize
+        }
+      });
+
+      const root = create(
+        <html.div style={[theme, styles.root]}>
+          <html.div style={styles.text}>text</html.div>
+        </html.div>
+      );
+      expect(root.toJSON()).toMatchSnapshot();
+    });
```

Comment:
This is the new regression test for the React Native crash. It also regression tests the fix to inherited `lineHeight` that was found to affect auto-wrapped text.